aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md

```diff
@@ -30,27 +30,27 @@ At the end of this tutorial, you will have an MVC application with the ASP.NET I
 1. Log in to the [Azure Portal](https://go.microsoft.com/fwlink/?linkid=529715&amp;clcid=0x409).
 2. Click **NEW** at the bottom of the page, and then select **STORE**:
 
-    [![Displays menu with STORE entry selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.png)
+    [![Screenshot of Azure Portal menu with STORE entry selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.png)
```

Comment:
Too short; consider rewriting to: "Screenshot of the Azure Portal menu with the Store menu item highlighted with a red rectangle."